lib/utils/logger.ts

```diff
@@ -63,8 +63,10 @@ class Logger {
 		};
 
 		_.forEach(this.streams, function(stream, key) {
-			if (key !== 'debug' || process.env.DEBUG) {
+			if (key !== 'debug') {
 				stream.pipe(process.stdout);
+			} else if (process.env.DEBUG) {
+				stream.pipe(process.stderr);
```

Comment:
Good! Arguably I think all logging output (whatever the log level: info, warn, debug, error...) should always have been printed to stderr, and only "application output" (like the output of `balena version` or `balena envs`) should be printed to stdout. But such change would break compatibility and would require a major version bump. I've added this note for future consideration in the GDoc: [CLI - breaking changes for next major release](https://docs.google.com/document/d/1uHE-_LLu5gJUQZICQJtlo76EQCm90z0CMAwpkfw6s8w/edit?usp=sharing)